HW_3/templates/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HW_3</title>
+</head>
+<body>
+<h1>Регистрация</h1>
+<form method="POST" action="{{ url_for('submit') }}">
+{{ form.csrf_token }}
+<p>
+{{ form.username.label }}<br>
+{{ form.username() }}
+</p>
+    <p>
+{{ form.user_surname.label }}<br>
+{{ form.user_surname() }}
+</p>
+<p>
+{{ form.email.label }}<br>
+{{ form.email() }}
+</p>
+    <p>
+{{ form.password.label }}<br>
+{{ form.password() }}
+</p>
+    <p>{{form.submit()}}</p>
```

Comment:
А тут много повторений. Yужно привести код в соответствии с принципом DRY
```suggestion
<p>
    {% for field in form if field.name != 'csrf_token' %}
    <p>
        {{ field.label }}<br>
        {{ field }}
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
    </p>
    {% endfor %}
    <p>
```
Причем строки
```html
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
```
будут повторять flash сообщение об ошибке. Повторюсь, что вы можете решить, что лучше: либо выводить сообщение тут, либо выше.